webpack.prod.js

```diff
@@ -3,6 +3,7 @@ const path = require('path');
 const webpack = require('webpack');
 const merge = require('webpack-merge');
 const ExtractTextPlugin = require('extract-text-webpack-plugin');
+const UglifyJSPlugin = require('uglifyjs-webpack-plugin');
```

Comment:
Should be `UglifyJsPlugin` (lowercase "s").

Reply:
Yup, already figured that out, thank you. This is throwing some really weird errors about components not being mounted (after this line is fixed), please check it out if you get a chance.